app/src/main/java/de/danoeh/antennapod/activity/CastEnabledActivity.java

```diff
@@ -38,7 +39,6 @@ protected void onCreate(Bundle savedInstanceState) {
         PreferenceManager.getDefaultSharedPreferences(getApplicationContext()).
                 registerOnSharedPreferenceChangeListener(this);
 
-        castUICounter = 0;
         mCastManager = CastManager.getInstance();
         mCastManager.addCastConsumer(castConsumer);
         isCastEnabled = UserPreferences.isCastEnabled();
```

Comment:
unlike when the preference is changed, here we're sure that `isResumed == false`, so no locking is needed